.vscode/settings.json

```diff
@@ -11,5 +11,9 @@
 		"editor.defaultFormatter": "esbenp.prettier-vscode"
 	},
 
-	"dotnet.defaultSolution": "backend/FactoryOps.sln"
+	"dotnet.defaultSolution": "backend/FactoryOps.sln",
+	"cSpell.words": [
+		"HSTM",
+		"Manzanillo"
```

Comment:
And what is it? Why is manzanillo served?

Reply:
it's for checking spelling